modules/basic-skills/src/actions/call_api.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios')
+
+/**
+ * @hidden true
+ */
+const callApi = async (url, method, body, memory, variable) => {
```

Comment:
I think the axios call should wrapped in a try/catch so errors can be handled more easily